### PR TITLE
Code that does not work for exercise 1.7 Mantra

### PR DESCRIPTION
**Pull request by krachamallu**
@twang 
Attempts to print out the following using a method to avoid repetition

There's one thing every coder must understand:
The System.out.println command.

There's one thing every coder must understand:
The System.out.println command.

### CODE REVIEW

File: Mantra.java [hunk 1]
```diff
@@ -0,0 +1,10 @@
+public class Mantra {
+    public static void main(String[] args) {
+        printer; 
+    }
```
Comment (rsjrsj) [inline Mantra.java]: You want to print this twice, so run the method "printer" twice.

Comment (twang) [reply Mantra.java]: Also, how do we call a method?

**Comment by twang** [comment]
Thanks @krachamallu and @rsjrsj!

File: Mantra.java (hunk 1 above)
Comment (rsjrsj) [reply Mantra.java]: printer();?

**Pull request closed without merging**